### ISSUE TITLE
Help me to install on windows

### ISSUE THREAD

**Issue by soundarps** [HaikuTeam/animator#1062]
Hey am a designer who is trying to create a Lottie file I really can't understand how to download your haiku animator app this is frustrating I tried reading that Windows OS dependencies I can't understand anything also I tried searching for installation in youtube and google I can't find anything useful. Help me

**Comment by jayphee18** [comment]
Yeah, I am also a web designer and I want to expand my skills in web design so I want to learn more and make Lottie animations for my profile website first then I will use it for my clients. Please make a guide about how to install as I am also not familiar with the process. 

I tried installing Powershell on my Windows PC pasted the install commands. I don't know what to do after.

**Comment by lsuhadolnik** [comment]
Windows build is doomed :/
I have put a whole day into trying to build a working installer, but I didn't succeed.

I have followed the instructions.
The `yarn  electron-rebuild` fails everytime due to @ronomon/base64 package failing to build. MSBuild does not find the Windows.h file and some other headers.

Does someone know how to build this project by hand?



When I run `generate_windows_installer.bat`, the build succeeds after removing environment variable checks inside `notarize.js` and `deploy.js`.

When I install the built installer, it runs and installs everything - electron, all DLLs which were previously found in the paid version, proper folder structure, app.ASAR...
But when I run the exe, it spawns 3 processes which all die in 5 seconds.

Could someone guide me about how to enable logging?


Meanwhile: Linux installation instructions work like a charm.

**Comment by baltao1** [comment]
"Apple INC" bribed "Haiku For Teams" for making it hard and a lot like looking into the oblivion for Windows users.
Linux guys be like :- (After launching Haiku Animator faster than MAC users installing the .dmg file) HEHEHEHEHEHE
